composer.json

```diff
@@ -12,15 +12,15 @@
         }
     ],
     "require": {
-        "php": ">=5.6.4",
+        "php": ">=7.1",
```

Comment:
It will be great for us not to skip 7.0. A lot of folks got on to 7.0 and they haven't upgraded yet.

Reply:
Fair point, happy to drop back to 7.0 (even though it's out of support)

Reply:
Yeah. Sometime around end of Q1, we can drop it totally.